src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala

```diff
@@ -99,12 +99,11 @@ class ApiValidationSpec
   it should "ensure that all admin routes have extra auditing enabled" in {
     synchRequest(ValidSynchRequest.payload) ~> Route.seal(createRoutesFromDerivations(ingressDerivationLogic)) ~> check {
       val ingressii = responseAs[TestSynchResponse].ingressii
-      ingressii
-        .filter(_.name.contains("jblogs"))
-        .map(_.filters.get)
-        .foreach { filterChain =>
-          filterChain should include("""enableAccessLog() -> unverifiedAuditLog("https://identity.zalando.com/managed-id")""")
```

Comment:
How was this passing previously? There was nothing in the filter?

Reply:
Yeah the list was empty

Reply:
😭 
Shoddy testing....

Reply:
Cowboys Ted!